### DISCUSSION TITLE
Guestbook

### DISCUSSION THREAD

**Discussion by rainbowdashlabs** [rainbowdashlabs/rainbowdashlabs#1] [General]
Hello stranger, fellow contributor or friend!

Welcome to my guestbook. In case you like the stuff I am doing feel free to leave a nice message. If you want you can also greet your grandparents, btw you should call them again.

However please be nice and friendly. We all know that the internet is such a peaceful place UwU

**Comment by Taucher2003** [comment]
I just want to leave a message

**Comment by warriorzz** [comment]
a nice message
jokes aside, thank you for your awesome contributions

**Comment by NyCodeGHG** [comment]
i like your projects

**Comment by Skyslycer** [comment]
i like... LANDJÄGER

**Comment by Paul2708** [comment]
You are awesome! ❤️

**Comment by kxmpxtxnt** [comment]
Ich liebe Kekse

**Comment by Goldmensch** [comment]
I like your nice way of chatting with people, making fun and helping people. Unfortunately, there are far too few nice and helpful people in this world. You are a real enrichment for the world. 

Stay healthy and have a nice day :3

**Comment by cancel-cloud** [comment]
I just wanted to say something nice, so here is a "nice" ^^

Cool work dude <3

**Comment by victorpyra** [comment]
am i now a member of the council??
Nice work :)

**Comment by yannicklamprecht** [comment]
UWU

**Comment by LuftigerLuca** [comment]
⣿⣿⣿⠟⠛⠛⠻⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡟⢋⣩⣉⢻
⣿⣿⣿⠀⣿⣶⣕⣈⠹⠿⠿⠿⠿⠟⠛⣛⢋⣰⠣⣿⣿⠀⣿
⣿⣿⣿⡀⣿⣿⣿⣧⢻⣿⣶⣷⣿⣿⣿⣿⣿⣿⠿⠶⡝⠀⣿
⣿⣿⣿⣷⠘⣿⣿⣿⢏⣿⣿⣋⣀⣈⣻⣿⣿⣷⣤⣤⣿⡐⢿
⣿⣿⣿⣿⣆⢩⣝⣫⣾⣿⣿⣿⣿⡟⠿⠿⠦⠀⠸⠿⣻⣿⡄⢻
⣿⣿⣿⣿⣿⡄⢻⣿⣿⣿⣿⣿⣿⣿⣿⣶⣶⣾⣿⣿⣿⣿⠇⣼
⣿⣿⣿⣿⣿⣿⡄⢿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡟⣰
⣿⣿⣿⣿⣿⣿⠇⣼⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⢀⣿
⣿⣿⣿⣿⣿⠏⢰⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⢸⣿
⣿⣿⣿⣿⠟⣰⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⠀⣿
⣿⣿⣿⠋⣴⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡄⣿
⣿⣿⠋⣼⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡇⢸

**Comment by jqshuv** [comment]
I was here...

**Comment by AljoshaVieth** [comment]
Hello there!

long time no see

**Comment by PokeFred** [comment]
⣿⣿⣿⠟⠛⠛⠻⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡟⢋⣩⣉⢻
⣿⣿⣿⠀⣿⣶⣕⣈⠹⠿⠿⠿⠿⠟⠛⣛⢋⣰⠣⣿⣿⠀⣿
⣿⣿⣿⡀⣿⣿⣿⣧⢻⣿⣶⣷⣿⣿⣿⣿⣿⣿⠿⠶⡝⠀⣿
⣿⣿⣿⣷⠘⣿⣿⣿⢏⣿⣿⣋⣀⣈⣻⣿⣿⣷⣤⣤⣿⡐⢿
⣿⣿⣿⣿⣆⢩⣝⣫⣾⣿⣿⣿⣿⡟⠿⠿⠦⠀⠸⠿⣻⣿⡄⢻
⣿⣿⣿⣿⣿⡄⢻⣿⣿⣿⣿⣿⣿⣿⣿⣶⣶⣾⣿⣿⣿⣿⠇⣼
⣿⣿⣿⣿⣿⣿⡄⢿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡟⣰
⣿⣿⣿⣿⣿⣿⠇⣼⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⢀⣿
⣿⣿⣿⣿⣿⠏⢰⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⢸⣿
⣿⣿⣿⣿⠟⣰⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⠀⣿
⣿⣿⣿⠋⣴⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡄⣿
⣿⣿⠋⣼⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡇⢸

**Comment by TheMeinerLP** [comment]
Nice to see you here :D

**Comment by InsomnioraDE** [comment]


**Reply by rainbowdashlabs** [reply]
Hello Kevin. Both servers having the same IPv4 is technically not possible c: unless you have an IPv6 only server but even then should you not have the same IPv4 as my server since I have a dedicated IPv4 assigned 

The server which is currently running under the ip associated with eldoria.de is a dedicated server which houses all my infrastructure. So no it's not possible to shut it down.

Also eldoria was never published and when I wrote about it I used my domain. So I don't know how you should end up with one of my previous ips. 

And eldoria is not dead. Its the brand I publish and sell minecraft plugins under

**Reply by InsomnioraDE** [reply]
![image](https://github.com/rainbowdashlabs/rainbowdashlabs/assets/150966574/3b96d214-d782-4cde-b47b-f057cc7f5ff8)

It actually shows my Icon and your motd. When I enter the server I almost land half of the time on your server.
That's why I texted you

**Reply by NyCodeGHG** [reply]
that sounds a lot like your configured dns server is broken.
@rainbowdashlabs' server is hosted at hetzner, while your server seems to be running at contabo, so this is probably also not a routing issue

**Reply by rainbowdashlabs** [reply]
I dont even have a minecraft server running that has this kind of motd.
I have two servers running. And the only one eldoria related is demo.eldoria.de.
Maybe you just mixed up your settings...

**Reply by rainbowdashlabs** [reply]
Also both motd and the base64 encoded server image are contained inside the [status response](https://wiki.vg/Server_List_Ping#Status_Response), so there is no way this could come from two different servers.

Or contact me on discord and send me your detailed dns settings. I still cant believe how this should be possible

**Reply by rainbowdashlabs** [reply]
Also dont know on which server you think you are joining. There is no minecraft server you could join with entering my servers ip because I route you to the correct server based on the used subdomain. You would simply run into an error when doing this. The last time someone connected on my demo server was 3 month ago and it was me.

All I see in my router are connection tries on eldoria.de itself that are denied of course. You cant join any server with that. I see no tries at all on the ip itself. or anything not eldoria related, which would be the case if people would join of your domain with my ip behind it.

**Comment by drakon64** [comment]
uwu

**Comment by Strokkur424** [comment]
Hey o/
I found you by chance on discord lol!
Keep up the great work on whatever you are coding actively at the moment! I am sure it is awesome!
Just leaving you a positive comment, because I am sure it will make you happy for at least 5 seconds :D

**Reply by rainbowdashlabs** [reply]
It surely did make me happy :D
